src/main/java/org/folio/sidecar/service/routing/RequestForwardingService.java

```diff
@@ -49,13 +61,16 @@ public class RequestForwardingService {
    */
   @SneakyThrows
   public void forward(RoutingContext rc, String absUri) {
+    forward(rc, absUri, webClient);
+  }
+
+  private void forward(RoutingContext rc, String absUri, WebClient webClient) {
```

Comment:
Private method is above on public method

Reply:
Right, I just had a checkstyle violation. I will suppress it then